### DISCUSSION TITLE
Spoiler Labels: Change Button Variant

### DISCUSSION THREAD

**Discussion by cohogan** [mantinedev/mantine#5809] [Q&A] [answered]
I'm using the Spoiler component in my application, but I'd like to change the appearance of the showLabel and hideLabel. As far as I can tell, these props just are added as the label of a button which controls the show and hide behavior. Is there any way to change the props of this button? For instance, if I wanted to make the button fullWidth, change the margins, or change the button variant?

Thank you so much for the help. I love this component library.

**Comment by rtivital** [comment, marked as answer]
You can change Spoiler component styles with Styles API
